FunctionApp/flex-dotnet-functionapp-on-azure.yml

```diff
@@ -0,0 +1,91 @@
+name: Build and deploy .NET project to Azure Function App
```

Comment:
I assume this is specific to Flex Consumption given the file name - should this say Flex Consumption somewhere in this name? This comment applies to all the other yaml files in this PR

Reply:
Yea, we could. The other workflows don't mention the SKU, but I can see value in plugging "Flex Consumption" to increase the visibility of the product